### PR TITLE
Bug 1833886 - Update guide to merging contributor PRs doc

### PR DESCRIPTION
**Pull request by rocketsroger**
### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/shared/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Breaking Changes**: If this is a breaking Android Components change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-apk-{fenix,focus,klar}-debug` task you're interested in.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1833886

### CODE REVIEW

**Review by MatthewTighe — APPROVED** [review]
LGTM, thanks for doing this 🙏

File: fenix/docs/Guide-to-merging-contributor-PRs.md [hunk 1]
```diff
@@ -1,52 +1,22 @@
...
-**Once you create this PR, the CI for both the original and the duplicate PRs will run. When everything is green, you can merge either of them.**
-
-5. To land the duplicate, close the original PR first, refresh mergify (`@Mergifyio refresh`), and then add `needs-landing` label to your PR.
-
-* Mergify won’t merge the duplicate while the original is open, since they both have the same SHA and mergify does honour the first one over those created consequently.
+3. Review the code to make sure everything is clean.
 
-OR
+4. Once a Firefox Android team member has reviewed the PR and deemed it safe, comment the following to start UI tests.
+```bors try```
```
[the remark this answers is not in the document]
Comment (MatthewTighe) [reply fenix/docs/Guide-to-merging-contributor-PRs.md:18]: On [this patch](https://github.com/mozilla-mobile/firefox-android/pull/1727) I have only been able to get it to work without the `@`, actually.

File: fenix/docs/Guide-to-merging-contributor-PRs.md [hunk 2]
```diff
@@ -1,52 +1,22 @@
...
 
-## Command line instructions
-
-_Note: these instructions use https://cli.github.com/_
-
-1. Fetch upstream changes and locally check out the contributor's branch onto your fork
+1. Fetch upstream changes and locally check out the contributor's branch onto your fork.
 ```sh
 git fetch --all
 gh pr checkout <PR number>` or `gh pr checkout <PR number> -R mozilla-mobile/fenix
```
Comment (gabrielluong) [inline fenix/docs/Guide-to-merging-contributor-PRs.md:8]: Perhaps we could just simplify this to be just 
gh pr checkout <PR number>

I have only ran the command (without the fetch) that I see from the following:
Screen Shot 2023-05-18 at 1 04 39 PM

File: fenix/docs/Guide-to-merging-contributor-PRs.md [hunk 3]
```diff
@@ -1,52 +1,22 @@
...
 
-OR
+4. Once a Firefox Android team member has reviewed the PR and deemed it safe, comment the following to start UI tests.
+```bors try```
 
-5. To land the original:
-* i. Make sure that contributor's branch hasn't diverged from yours (they must have the same SHA).
-* ii. The change has to be on the top of the main branch when it is first in line in the merge queue.
-* iii. It requires the needs-landing label.
+5. Once the UI tests have all completed and passed, remove `needs review` and add `approved` and `needs landing` label the contributor's PR.
```
Comment (gabrielluong) [inline fenix/docs/Guide-to-merging-contributor-PRs.md:20]:
```suggestion
5. Once the UI tests have all completed and passed, add `needs landing` label the contributor's PR.
```
We won't need to manually remove labels after my GitHub actions land.

File: fenix/docs/Guide-to-merging-contributor-PRs.md [hunk 4]
```diff
@@ -1,52 +1,22 @@
 # Guide to merging contributor PRs for Firefox Android team members
```
Comment (gabrielluong) [inline fenix/docs/Guide-to-merging-contributor-PRs.md:1]: Perhaps also add instructions on how to edit and push changes for contributor PR as well?

**Review by gabrielluong — COMMENTED** [review]
No need to wait for further approval from me.

File: fenix/docs/Guide-to-merging-contributor-PRs.md (hunk 2 above)
Comment (rocketsroger) [reply fenix/docs/Guide-to-merging-contributor-PRs.md:8]: Sure, I'll update. thanks

File: fenix/docs/Guide-to-merging-contributor-PRs.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (rocketsroger) [reply fenix/docs/Guide-to-merging-contributor-PRs.md:18]: I will try "@bors try" on that patch to see if that works as well

File: fenix/docs/Guide-to-merging-contributor-PRs.md (hunk 4 above)
Comment (rocketsroger) [reply fenix/docs/Guide-to-merging-contributor-PRs.md:1]: Sure, I'll do this as a follow up.  It'll be more complicated.

File: fenix/docs/Guide-to-merging-contributor-PRs.md (hunk 3 above)
Comment (rocketsroger) [reply fenix/docs/Guide-to-merging-contributor-PRs.md]: Sure, I'll remove

**Pull request merged**